### PR TITLE
MHV 53639 Secure Messaging content for North Chicago transition to Cerner 

### PR DESCRIPTION
**Pull request by alex-morgun-1**
## Summary

SM on VA.gov Implement the pre-transition content logic for OH transitions T-30

February 6, 1 AM CT

This includes research and implementation

User Story:  As a Veteran users of a transitioning facility, I want to have limited functionality on messages associated with the transitioning facility so that I do not accidentally send messages that could be lost.

- any SM users that have associations with North Chicago facility station# 556 should start seeing a warning banner notifying them about the upcoming transition of EHR to Cerner
- content should be displayed on My Health SM Landing page and My Health SM Inbox page
-  this functionality is behind a feature flag `cerner_transition_556_t30`, which will be toggled on T-30 as per requirements. The same feature flag should be used for testing in RI 
-  ~~❗ pending final content version~~ Updated content per https://github.com/department-of-veterans-affairs/va.gov-team/issues/74052

## Related issue(s)

- https://jira.devops.va.gov/browse/MHV-53639

## Testing done

- _Describe what the old behavior was prior to the change_
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Desktop | ||

## What areas of the site does it impact?

My Health - Secure Messaging

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Comment by oleksii-morgun-va** [comment]
this will require toggling feature flags in RI. This can be done by navigating to `http://{REPLACE_WITH_REVIEW_INSTANCE_ID}-api.review.vetsgov-internal/flipper/features` like this example here `http://b820bbd80f6e771550174b22845c44d2-api.review.vetsgov-internal/flipper/features`

**Review by KyleCardwell — DISMISSED** [review]
Looks good

**Review by KyleCardwell — DISMISSED** [review]
LGTM!

**Review by IGallupSoCo — DISMISSED** [review]
LGTM

File: src/applications/mhv/secure-messaging/components/MessageList/CernerFacilityAlert.jsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ const CernerFacilityAlert = () => {
     <>
       {cernerFacilities?.length > 0 && (
         <va-alert
-          className="vads-u-margin-bottom--2"
+          class="vads-u-margin-bottom--2"
```
Comment (vsaleem) [inline src/applications/mhv/secure-messaging/components/MessageList/CernerFacilityAlert.jsx:29]: Should this be className instead?

Comment (oleksii-morgun-va) [reply src/applications/mhv/secure-messaging/components/MessageList/CernerFacilityAlert.jsx:29]: I find that `className` has no effect for some/most VA Design library components

**Review by KyleCardwell — APPROVED** [review]
Approved

**Review by vsaleem — APPROVED** [review]

**Review by IGallupSoCo — APPROVED** [review]
LGTM

**Pull request merged**